tests/test_idutils.py

```diff
@@ -32,25 +32,38 @@
     ('http://www.example.org/ark:/13030/tqb3kh97gh8w', ['ark', 'url'], '', ''),
     ('10.1016/j.epsl.2011.11.037', ['doi', 'handle'],
         '10.1016/j.epsl.2011.11.037',
-        'https://doi.org/10.1016/j.epsl.2011.11.037'),
+        'https://doi.org/10.1016%2Fj.epsl.2011.11.037'),
```

Comment:
Is this correct according to the DOI spec? I think all ascii characters should probably stay as is.

Reply:
According to [this part of the spec](https://www.doi.org/doi_handbook/2_Numbering.html#2.6.3), (and the final sentence of [this section](https://www.doi.org/doi_handbook/2_Numbering.html#2.5.2.4)), it's better for forward slashes to be url-encoded

Reply:
This is funny - so [2.5.2.4](https://www.doi.org/doi_handbook/2_Numbering.html#2.5.2.4) puts slash in the recommend category for encoding. Just below in [2.6.2](https://www.doi.org/doi_handbook/2_Numbering.html#2.6.2) it says:

> The DOI name "10.1006/jmbi.1998.2354" would be made an actionable link as "https://doi.org/10.1006/jmbi.1998.2354".

(i.e. no encoding) :S

Let's discuss IRL - not fully sure what the impact would be.